tomcat/src/main/java/org/apache/coyote/http11/HttpRequest.java

```diff
@@ -35,11 +70,27 @@ public ContentType getContentType() {
     }
 
     public String getQueryParameterValue(String key) {
-        String value = queryParameter.get(key);
-        if (value == null || value.isBlank()) {
-            return "";
-        }
-        return value;
+        return queryParameter.getOrDefault(key, "");
+    }
+
+    public boolean hasCookie(String key) {
+        HttpCookie httpCookie = cookies.stream()
+                .filter(cookie -> cookie.isKey(key))
+                .findFirst()
+                .orElse(null);
+        return httpCookie != null;
```

Comment:
anyMatch 사용하면 더 간결해질 것 같아요.
```suggestion
        return cookies.stream()
            .anyMatch(cookie -> cookie.isKey(key));
```

Reply:
오옷 천재?
반영했습니다.